### PR TITLE
docs: update examples in `math/base/special/lcmf`

### PR DESCRIPTION
**Pull request by gunjjoshi**
## Description

> What is the purpose of this pull request?

This pull request:

-   updates JS examples in [`math/base/special/lcmf`](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/math/base/special/lcmf) to use [`@stdlib/random/array/discrete-uniform`](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/random/array/discrete-uniform).
-   addresses suggestions on https://github.com/stdlib-js/stdlib/commit/c0d083d4329c439d9c002479740518584e5f5977.

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves https://github.com/stdlib-js/todo/issues/2318

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

**Comment by Planeshifter** [comment]
@gunjjoshi You mind simplifying the example code further, please

```js
var a = randu( 100, 0, 50 );
var b = randu( 100, 0, 50 );
var i;
```

?

**Review by Planeshifter — APPROVED** [review]
Thanks, @gunjjoshi!

**Comment by gunjjoshi** [comment]
> @gunjjoshi You mind simplifying the example code further, please
> 
> ```js
> var a = randu( 100, 0, 50 );
> var b = randu( 100, 0, 50 );
> var i;
> ```
> 
> ?

Done, thanks for the suggestion!

File: lib/node_modules/@stdlib/math/base/special/lcmf/README.md [hunk 1]
```diff
@@ -101,20 +101,15 @@ v = lcmf( 48, NaN );
...
-var roundf = require( '@stdlib/math/base/special/roundf' );
+var randu = require( '@stdlib/random/array/discrete-uniform' );
 var lcmf = require( '@stdlib/math/base/special/lcmf' );
 
-var a;
-var b;
-var v;
+var a = randu( 100, 0, 50 );
+var b = randu( 100, 0, 50 );
 var i;
```
Comment (kgryte) [inline lib/node_modules/@stdlib/math/base/special/lcmf/README.md]:
```suggestion

var i;
```

File: lib/node_modules/@stdlib/math/base/special/lcmf/examples/index.js [hunk 2]
```diff
@@ -18,18 +18,13 @@
...
-var roundf = require( '@stdlib/math/base/special/roundf' );
+var randu = require( '@stdlib/random/array/discrete-uniform' );
 var lcmf = require( './../lib' );
 
-var a;
-var b;
-var v;
+var a = randu( 100, 0, 50 );
+var b = randu( 100, 0, 50 );
 var i;
```
Comment (kgryte) [inline lib/node_modules/@stdlib/math/base/special/lcmf/examples/index.js]:
```suggestion

var i;
```

**Review by kgryte — APPROVED** [review]
LGTM. Thanks, @gunjjoshi and @Planeshifter!

**Pull request merged**